how-to/customize-workspace/public/manifest.fin.json

```diff
@@ -882,7 +882,7 @@
 				{
 					"enabled": true,
 					"id": "developer-actions",
-					"url": "http://localhost:8080/js/modules/actions/developer.bundle.js"
+					"url": "http://localhost:8080/js/modules/composite/developer.bundle.js"

```

Comment:
We probably need a way to switch off developer bundle in Sales Studio, I noticed when Jesper was demoing the menu entries appear, which looks kind of rubbish